transports/quic/Cargo.toml

```diff
@@ -26,6 +26,17 @@ tokio = { version = "1.35.1", default-features = false, features = ["net", "rt",
 tracing = "0.1.37"
 socket2 = "0.5.5"
 ring = "0.16.20"
+sha2 = "0.10.7"
+hex = "0.4"
+
+# WebTransport
+http = "1.0.0"
+h3 = { git = "https://github.com/hyperium/h3" }
+h3-quinn = { git = "https://github.com/hyperium/h3" }
+h3-webtransport = { git = "https://github.com/hyperium/h3" }
```

Comment:
With git versions, everything is ok:

<img width="794" alt="image" src="https://github.com/libp2p/rust-libp2p/assets/25565200/284198b3-97b7-4811-aeca-098acd35de8b">

But when I define `h3 = "0.0.4"` I got two versions `0.0.3` and `0.0.4`:

<img width="807" alt="image" src="https://github.com/libp2p/rust-libp2p/assets/25565200/0ab5b157-2098-4a45-b97e-2d44779627d8">


And, accordingly, I got a lot of errors.
I stuck with this problem for the first time, do you know how to fix it?

Reply:
It is because there hasn't yet been a release of `h3-webtransport` with `h3 0.0.4`. See https://crates.io/crates/h3-webtransport/0.1.0/dependencies. I'd suggest nudging the maintainers for a new release!

Reply:
OMG
Thanks, I'll do it a bit later